modules/timepoint_list/README.md

```diff
@@ -0,0 +1,48 @@
+# Timepoint List
+
+## Purpose
+
+The `timepoint_list` module serves as an entry-point for data entry
+about a candidate. It serves as a dashboard for a candidate linking
+to other modules that provide specific functionality about a candidate.
+
+## Intended Users
+
+The `timepoint_list` module is primarily intended for data entry
+staff who are accessing a candidate.
+
+## Scope
+
+The `timepoint_list` module provides a mechanism to link to
+information about candidates. 
+
```

Comment:
```suggestion
In particular, it provides access to longitudinally-organized behavioural data, and links to Imaging data. It is also the only access point for Candidate Parameters, where Consent is entered as well as other information relevant to the candidate across timepoints. 
```

Reply:
I don't see why it would go into either.